sound/soc/sof/intel/pci-tgl.c

```diff
@@ -84,6 +84,22 @@ static const struct sof_dev_desc adls_desc = {
 	.ops = &sof_tgl_ops,
 };
 
+static const struct sof_dev_desc adl_desc = {
+	.machines               = snd_soc_acpi_intel_adl_machines,
+	.alt_machines           = snd_soc_acpi_intel_adl_sdw_machines,
+	.resindex_lpe_base      = 0,
+	.resindex_pcicfg_base   = -1,
+	.resindex_imr_base      = -1,
+	.irqindex_host_ipc      = -1,
+	.resindex_dma_base      = -1,
+	.chip_info = &tgl_chip_info,
```

Comment:
Should we add a comment like ` /* Alderlake */` in `const struct sof_intel_dsp_desc tgl_chip_info = {}` if it is for ADL, too?

Reply:
sure. will do